src/OrchardCore.Modules/Orchard.Liquid/Handlers/LiquidPartHandler.cs

```diff
@@ -9,31 +10,24 @@ namespace Orchard.Liquid.Handlers
     public class LiquidPartHandler : ContentPartHandler<LiquidPart>
     {
         private readonly IContentDefinitionManager _contentDefinitionManager;
-        private readonly ILiquidManager _liquidManager;
 
-        public LiquidPartHandler(
-            IContentDefinitionManager contentDefinitionManager,
-            ILiquidManager liquidManager
-            )
+        public LiquidPartHandler(IContentDefinitionManager contentDefinitionManager)
         {
             _contentDefinitionManager = contentDefinitionManager;
-            _liquidManager = liquidManager;
         }
 
         public override void GetContentItemAspect(ContentItemAspectContext context, LiquidPart part)
         {
             context.For<BodyAspect>(bodyAspect =>
```

Comment:
No because `GetContentItemAspect` is not async.
That doesn't matter right now as I create this https://github.com/OrchardCMS/Orchard2/issues/822 which will solve it.